department-for-transport-BODS/bods
# round the data quality score to lower number instead of higher rounded number

Testing instructions

- Publish a timetable and view the data quality score in the review page
- Access below page and verify the data quality score
  Publish endpoint -> Review Published Dataset -> Select a published timetable
